content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,237 @@
-- **Discounted Product**: allows you to create one barcode per applied discount. You can then scan your product in the POS and then scan the discount barcode, discount will be applied on the normal price of the product.
-- **Weighted Product**: allows you to identify the product and specify its weight, used in both POS (in which the price is computed based on the weight) and in inventory.
-- **Client**: allows you to identify the customer, for example used with loyalty program.
-- **Cashier**: allows you to identify the cashier when entering the POS.
-- **Location**: allows you to identify the location on a transfer when multi-location is activated.
-- **Package**: allows you to identify packages on a transfer when packages are activated.
-- **Credit Card**: doesn’t need manual modification, exists for data from the Mercury module.
-- **Unit Product**: allows you to identify a product for both POS and transfers.
+To use default nomenclature, navigate to :menuselection:`Inventory app --> Configuration -->
+Settings`. Under the :guilabel:`Barcode` section, tick the :guilabel:`Barcode Scanner` checkbox.
+Doing so installs the **Barcode** app in the database.
+
+Next, in the :guilabel:`Barcode Nomenclature` field, ensure :guilabel:`Default Nomenclature` is
+selected. Then, click :guilabel:`Save`.
+
+.. image:: barcode_nomenclature/enable-nomenclature.png
+   :align: center
+   :alt: Enabled barcode setting with Default Nomenclature selected.
+
+With the **Barcode** module installed, and the :guilabel:`Default Nomenclature` selected, the
+barcode actions using |UPC| and |EAN|, detailed in the :ref:`default nomenclature list
+<barcode/operations/default-nomenclature-list>`, are available for use. And, by default, Odoo
+automatically handles |UPC|/|EAN| conversion.
+
+.. _barcode/operations/product-weight:
+
+Example: product weight barcode
+===============================
+
+To better understand how barcode nomenclature is used to identify products in Odoo, this example
+where product weight barcodes in |EAN| format are used to allow a :doc:`Point of Sale
+<../../../sales/point_of_sale>` business to automatically print barcodes, and calculate the price
+using the weight of the item.
+
+To set up barcodes for weighted products, the following rule is used:
+
++--------------------------+----------------------------------+------------------------------------------+
+|        Rule Name         |       Barcode Pattern            |              Field in Odoo               |
++==========================+==================================+==========================================+
+| Weighted Barcodes 3      | (21)....{NNDDD}                  | :guilabel:`Barcode` field on product form|
+| Decimals                 |                                  |                                          |
++--------------------------+----------------------------------+------------------------------------------+
```

Comment:
Suggestion to use a list table, only because it is easier to maintain :), Up to you!
```suggestion
.. list-table::
   :header-rows: 1

   * - Rule Name
     - Barcode Pattern
     - Field in Odoo
   * - Weighted Barcodes 3 Decimals
     - (21)....{NNDDD}
     - :guilabel:`Barcode` field on product form
```